1.12.0/openebs-operator-1.12.0-RC1.yaml

```diff
@@ -578,7 +578,7 @@ spec:
       serviceAccountName: openebs-maya-operator
       containers:
         - name: node-disk-operator
-          image: quay.io/openebs/node-disk-operator-amd64:0.7.0-RC1
+          image: openebs/node-disk-operator:0.7.0-RC1
           imagePullPolicy: IfNotPresent
```

Comment:
@akhilerm can all images be changed to docker.

Reply:
Done. @kmova